src/library/simplified/extension/page-name-javascript-functions.js

```diff
@@ -463,6 +463,15 @@ window.utag.isLXI = function(){
     return false;
 }
 
+window.utag.isLXPymt = function(){
+    if(pageName.indexOf("page.LX.Checkout.Payment") > -1 ){
+        b["isLXPymt"] = true;
+        b["pageType"] = "payment";
```

Comment:
Uppercased P as in Payment?  I was looking at the existing code here that uses "Payment".